tests/src/index.test.ts

```diff
@@ -115,6 +115,22 @@ describe.each(CONFIGURATIONS)(
       })
     })
 
+    test('Create a doc by specifying a name', async () => {
+      const docResult = await DOCUMENT_MANANGER.createDoc({
+        doc: 'mydoc123',
+      })
+
+      expect(docResult.doc).toBe('mydoc123')
+    })
+
+    test('Reject invalid doc name', async () => {
+      await expect(
+        DOCUMENT_MANANGER.createDoc({
+          doc: 'mydoc123!',
+        }),
+      ).rejects.toThrow('400')
+    })
+
```

Comment:
What should happen when creating a doc with a name that's already been used? Is that a 409 Conflict?

Reply:
I think it should just return the same document. This way document creation is idempotent.